contracts/token/ERC721/ERC721.sol

```diff
@@ -7,22 +7,7 @@ import "./ERC721Basic.sol";
  * @title ERC-721 Non-Fungible Token Standard, optional enumeration extension
  * @dev See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
  */
-contract ERC721Enumerable is ERC721Basic {
-
-  bytes4 private constant InterfaceId_ERC721Enumerable = 0x780e9d63;
-  /**
-   * 0x780e9d63 ===
-   *   bytes4(keccak256('totalSupply()')) ^
-   *   bytes4(keccak256('tokenOfOwnerByIndex(address,uint256)')) ^
-   *   bytes4(keccak256('tokenByIndex(uint256)'))
-   */
-
-  constructor()
-    public
-  {
-    _registerInterface(InterfaceId_ERC721Enumerable);
-  }
-
+contract ERC721Enumerable {
```

Comment:
Was removing the `ERC721Basic` parent intentional? If so, what is the rationale? (Same question for `ERC721Metadata`.)

Reply:
ah, it was not intentional thanks for catching that. made the mistake of removing both parents instead of just the 165 implementation one